db/seeds.rb

```diff
@@ -1,13 +1,21 @@
 require 'factory_girl'
 include FactoryGirl::Syntax::Methods
+require 'timecop'
 
 2.times { create :user, :admin }
 
 5.times { create :user }
 
-50.times { create :passenger, :permanent }
-50.times { create :passenger, :recently_expired }
-50.times { create :passenger, :expiring_soon }
-50.times { create :passenger, :expiration_overriden }
-50.times { create :passenger, :inactive }
+20.times { create :passenger, :permanent }
+20.times { create :passenger, :recently_expired }
+20.times { create :passenger, :expiring_soon }
+20.times { create :passenger, :expiration_overriden }
+20.times { create :passenger, :inactive }
 5.times  { create :passenger, :no_note }
+
+dispatchers = User.dispatchers
+30.times do
+  Timecop.freeze 1.month.ago + rand(1.month) do
```

Comment:
I think a comment would be good here, even if it is pretty clear it'll help us new programmers avoid confusion down the road.